.github/workflows/example-doc.yml

```diff
@@ -19,12 +19,16 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
+      - name: Build docker
+        run: |
+          docker build -t inara:edgiest .
+
       - name: Build draft PDF
         run: >-
           docker run \
               --volume "$(pwd):/data" \
               --user "$(id -u):$(id -g)" \
-              openjournals/inara:latest \
+              inara:edgiest \
```

Comment:
this just builds the docker image locally, so we can use it locally. this enables for the example build to reflect changes to the template, since if the template changes, then the one on dockerhub doesn't reflect it

Reply:
Added a comment in the config to explain this in 37b88ae

Reply:
(to be clear this was purely a question, am not a docker person, so i take no opinion on this/what is good to do/not good to do)